src/gui/GameController.java

```diff
@@ -99,10 +106,11 @@ public void updateGameState() {
     }
     
     /**
-     * Method to check if Cook and Player are in the same room.
+     * Method to check if Cook and Player are in the same room and if the player has a steak with or without ketchup.
      */
     public void inSameRoom() {
-        if (game.getPlayer().getCurrentRoom() == game.getCook().getCurrentRoom() && !game.getPlayer().hasItem("Steak")) {
+        if (game.getPlayer().getCurrentRoom() == game.getCook().getCurrentRoom() && !game.getPlayer().hasItem("Steak")
+                && !game.getPlayer().hasItem("Steak-with-ketchup")) {
```

Comment:
Hvis i newliner if sætningen som herunder er det nemmer at overskue.
```java
if (
    game.getPlayer().getCurrentRoom() == game.getCook().getCurrentRoom() &&
    !game.getPlayer().hasItem("Steak") && 
    !game.getPlayer().hasItem("Steak-with-ketchup")
) {
     gui.getStages().get("CookInteraction").show();
}
```

Reply:
fixed in: 5bfd32fc4f668a057efc1cbd6dcd8cc34d3e5f0a